Mock-api readme command and adding db.json

beatricehahn opened the pull request.

beatricehahn committed:
add db.json, update README with command to run mock API

beatricehahn committed:
deleted weird copy of db.json

scottmai commented on README.md:
```diff
@@ -80,11 +107,16 @@ https://git-scm.com/download/win
 - Install dependencies:
 `yarn install`
 
+- Install json-server:
+`yarn add json-server`
+
```
We don't actually need to have this in the README, since its reflected in the `package.json` file and will be installed whenever anyone runs `yarn install`

scottmai commented on frontend/db.json:
```diff
@@ -0,0 +1,65 @@
+    "meals": [
+        {
+            "name": "Breakfast", 
+            "date_time_start": "1/22/22",
+            "date_time_end": "1/22/22",
```
A couple of things here

- Can we call it "datetime_start" instead of "date_time_start" to match the API spec
- I forgot to specify the time format in the spec, its updated now. Can we use time in the format of "2022-01-24T23:46:25.872Z" ? You can easily get this by opening your browser console and typing `new Date().toISOString()` This is supposed to reflect when each meal will start and end so we'll need the specific time.

scottmai reviewed (changes requested):
Overall looks great, just a couple notes for the datetime_start and end

beatricehahn committed:
Update README.md

removed json-server installation command

beatricehahn committed:
Update db.json

changed spelling to match api spec by adding datetime to workplace settings

beatricehahn committed:
Update db.json

Re-editted time format to follow ISO 8601. The start and end dates are proxies.

beatricehahn committed:
Create settings.json

add "datetime" as valid vocabulary

scottmai commented on .vscode/settings.json:
```diff
@@ -0,0 +1,6 @@
+{
+    "cSpell.words": [
+        "datetime",
+        "Parkside"
+    ]
+}
```
how weirddd

scottmai reviewed (approved):
Beeeaaaautiful. This will allow us to start frontend development with ease 😎

beatricehahn committed:
these are not my changes but need to go through with this in order to correct a git issue

beatricehahn committed:
not my changes but need to commit them

beatricehahn committed:
restore db.json because I accidentally deleted the original and not a weird copy

scottmai commented on README.md:
```diff
@@ -80,11 +107,14 @@ https://git-scm.com/download/win
 `yarn start`
 
+- Run the mock API:
+`yarn json-server db.json -p 8000 --watch`
```
Does this command work for you? I think we might want `npx` instead of `yarn` here

scottmai reviewed (changes requested):
^

beatricehahn committed:
Update mock API instructions on readme

change 'yarn' to 'npx' for mock API running command instruction

scottmai reviewed (approved):
Amazing 🙌
